ingestion/tests/unit/test_query_parser.py

```diff
@@ -0,0 +1,94 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+Query parser utils tests
+"""
+import csv
+import json
+from datetime import datetime
+from unittest import TestCase
+
+from metadata.ingestion.api.workflow import Workflow
+from metadata.ingestion.models.table_queries import TableQuery
+from metadata.ingestion.processor.query_parser import QueryParserProcessor
+
+config = """
+{
+  "source": {
+    "type": "sample-usage",
+    "config": {
+      "database": "warehouse",
+      "service_name": "bigquery_gcp",
+      "sample_data_folder": "ingestion/tests/unit/resources"
```

Comment:
@ulixius9 @ayush-shah can we add complex queries to the query_log and see if they are passing the test

Reply:
ok, will add complex queries